src/node_modules/ringbuffer.js

```diff
@@ -0,0 +1,25 @@
+import {EventEmitter} from 'events';
+
+export class RingBuffer extends EventEmitter {
+	constructor(opts) {
+		super();
+		this.records = [];
+		this.limit   = opts.bufferSize;
+	}
+
+	write(log) {
+		this.records.push(log);
+		if (this.records.length > this.limit) {
```

Comment:
Good points @markstos, thanks for the review. 
We are actually emitting an event [here](https://github.com/rapid7/le_node/pull/122/files#diff-13bd19716a61073644f0cf6699227d6fR15) and we are listening to this in the Logger object that it belongs to. Then we log this event in internal logger of le_node once we start discarding events. Are you suggesting to emit another event for this in the Logger object itself here?